src/main/java/org/jenkinsci/plugins/docker/commons/DockerRegistryEndpoint.java

```diff
@@ -54,6 +71,34 @@ public DockerRegistryEndpoint(String url, String credentialsId) {
     }
 
     /**
+     * Parse the registry endpoint out of a registry:port/namespace/name:tag string. Credentials are set to the id
+     * passed.
+     * 
+     * @param s
+     * @param credentialsId
+     *            can be null
```

Comment:
Sorry, it was a typo. The initial message was supposed to have: "...add <code>@CheckForNull</code> annotation to the **parameter**..."